File: android/NotifyBridge/app/src/main/java/com/alpware/notifybridge/pairing/PairingPayload.kt
package com.alpware.notifybridge.pairing


/**
 * Contains the local connection details shared by the Mac app during QR pairing.
 */
data class PairingPayload(
    val host: String,
    val port: Int,
    val code: String,
    val name: String?,
    val fingerprint: String
)

File: macos/NotifyBridgeMac/NotifyBridgeMac/NotifyBridgeMac/PairingPayload.swift
//
//  PairingPayload.swift
//  NotifyBridgeMac
//
//  Created by Mahmut Alperen Ünal on 13.05.2026.
//

import Foundation

/// Contains the local connection details shared with the Android app during QR pairing.
struct PairingPayload: Codable {
    let type: String
    let host: String
    let port: Int
    let secret: String
    let name: String
}
